sky/skylet/providers/gcp/constants.py

```diff
@@ -9,6 +9,26 @@
     "mtu": 1460,
     "routingConfig": {"routingMode": "GLOBAL"},
 }
+# Required firewall rules for SkyPilot to work.
+FIREWALL_RULES_REQUIRED = [
+    {
+        "direction": "INGRESS",
+        "allowed": [
+            {'IPProtocol': 'tcp', 'ports': ['0-65535']},
+            {'IPProtocol': 'udp', 'ports': ['0-65535']},
+        ],
+        "sourceRanges": ["10.128.0.0/9"],
+    },
```

Comment:
Add a comment for this rule, such as `Allow internal connections among VMs on GCP to enable ray cluster to be set up in a multi-node cluster`?

Reply:
Added!